.github/workflows/docker.yml

```diff
@@ -39,8 +47,12 @@ jobs:
         id: docker_build
         uses: docker/build-push-action@v2
         with:
-          context: "./tools/docker_dev"
+          context: "."
           file: "./tools/docker_dev/Dockerfile"
           push: ${{ github.event_name != 'pull_request' }}
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache
           tags: |
-            scipy/scipy-dev:${{ steps.getrefs.outputs.date }}-${{ steps.getrefs.outputs.branch}}-${{ steps.getrefs.outputs.sha8 }}, scipy/scipy-dev:latest
\ No newline at end of file
+            scipy/scipy-dev:${{ steps.getrefs.outputs.date }}-${{ steps.getrefs.outputs.branch}}-${{ steps.getrefs.outputs.sha8 }}, scipy/scipy-dev:latest
+      - name: Image digest 
```

Comment:
What's a "digest" in this context? Maybe a comment about what this does would be useful.

Reply:
On it - basically prints out some details of the image being built/push - size, tag etc

Will add a note